cmd/internal/cli/plugin_inspect_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/app/singularity"
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+// PluginInspectCmd displays information about a plugin
+var PluginInspectCmd = &cobra.Command{
```

Comment:
Do you need to add:
```go
Args: cobra.ExactArgs(1),
DisableFlagsInUseLine: true,
```
*Or whatever cobra likes...*

Becuse you use `args[0]` in `singularity.InspectPlugin()`.

Reply:
This already exists a few lines down ;) `line 34`

Reply:
:+1: